ledger/src/shredder.rs

```diff
@@ -346,6 +369,38 @@ impl Shredder {
     }
 }
 
+impl ReedSolomonCache {
+    const CAPACITY: usize = 4 * DATA_SHREDS_PER_FEC_BLOCK;
```

Comment:
A comment on why this should be a function of `DATA_SHREDS_PER_FEC_BLOCK` would be helpful